app/models/planning_application.rb

```diff
@@ -215,6 +215,14 @@ def full_address
     "#{address_1}, #{town}, #{postcode}"
   end
 
+  def secure_change_url(local_authority, application_id, secure_token)
+    if ENV["RAILS_ENV"] == "production" || ENV["RAILS_ENV"] == "preview"
```

Comment:
We don't need this `if` statement any more, as both options are identical.

Reply:
I added it because localhost is http, not https

Reply:
ah! I missed that.